cmake/ext_java.cmake

```diff
@@ -2,6 +2,11 @@ include(FindJava)
 find_package(Java REQUIRED)
 include(UseJava)
 if (NOT ANDROID)
+    set(JAVA_AWT_LIBRARY NotNeeded)
+set(JAVA_JVM_LIBRARY NotNeeded)
```

Comment:
nit since indentation does not matter outside of scripts in cmake file but alignment?